creator-node/sequelize/migrations/20200623002237-add-parent-file-data.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+/**
+ * Adds two columns to the `files` table
+ * - fileName: the actual queryable source filename stripped of any prefixes
+ * - dirMultihash: if the file is in an IPFS directory, the CID/multihash for the parent dir
+ */
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction()
+    try {
+      await queryInterface.addColumn(
+        'Files',
+        'fileName',
+        {
+          type: Sequelize.TEXT,
+          allowNull: true
+        },
+        { transaction }
+      )
+      await queryInterface.addColumn(
+        'Files',
+        'dirMultihash',
+        {
+          type: Sequelize.TEXT,
+          allowNull: true
+        },
+        { transaction }
+      )
+
+      const files = (await queryInterface.sequelize.query(`SELECT * FROM "Files";`, { transaction }))[0]
+      for (let file of files) {
+        // Set the filename to the last uri "part" of the source file if present
+        // otherwise, use just the sourceFile
+        const fileName = file.sourceFile
```

Comment:
tested this and p sure it works on all sourceFile formats, but can you add a comment to that effect.
for sourceFile = '....mp3' it matches and returns full string.
what's purpose of the `|| file.sourceFile` though?

Reply:
Added a comment